app/presenters/publishing_api/worldwide_organisation_presenter.rb

```diff
@@ -48,6 +48,15 @@ def description
       item.summary
     end
 
+    def body
+      [
+        {
+          content_type: "text/govspeak",
+          content: item.body || "",
+        },
+      ]
+    end
```

Comment:
In some presenters, we just use the parsed govspeak as HTML directly as the body content (e.g. [`DetailedGuidePresenter`](https://github.com/alphagov/whitehall/blob/860cb32a8d71f28cf01f80c96bc83198d15d2190/app/presenters/publishing_api/detailed_guide_presenter.rb#L79-L81)), but in others we use a hash like this (e.g. [`RolePresenter`](https://github.com/alphagov/whitehall/blob/860cb32a8d71f28cf01f80c96bc83198d15d2190/app/presenters/publishing_api/role_presenter.rb#L81-L88)).  What's the reason for choosing this approach over sending the parsed govspeak downstream?

Reply:
Sorted!